src/programs/combine_stacks/combine_stacks.cpp

```diff
@@ -0,0 +1,117 @@
+#include "../../core/core_headers.h"
+#include <dirent.h>
+#include <regex>
+
+class
+CombineStack : public MyApp
+{
+
+	public:
+
+    void DoInteractiveUserInput();
+	bool DoCalculation();
+	
+
+	private:
+};
+
+IMPLEMENT_APP(CombineStack)
+
+// override the DoInteractiveUserInput
+
+void CombineStack::DoInteractiveUserInput()
+{
+
+	UserInput *my_input = new UserInput("CombineStack", 1.0);
+
+    std::string input_star_file	=		my_input->GetFilenameFromUser("Input Star File", "required star file", "starFile.star", true);  
+
+	std::string output_filename	=		my_input->GetFilenameFromUser("Output image file name", "the combined result", "output.mrc", false );
+
+
+	delete my_input;
+
+	my_current_job.Reset(2);
+	my_current_job.ManualSetArguments("tt", input_star_file.c_str(), output_filename.c_str());
+
+}
+
+bool CombineStack::DoCalculation()
+{
+    std::string input_star_file = my_current_job.arguments[0].ReturnStringArgument();
+	std::string	output_filename = my_current_job.arguments[1].ReturnStringArgument();
+    
+    MRCFile output_file(output_filename,true);
+
+    float input_pixel_size;
+    int input_pixel_size_set = 0;
+
+    BasicStarFileReader input_star_file_reader;
+	wxString star_error_text;
+
+	if (input_star_file_reader.ReadFile(input_star_file, &star_error_text) == false)
+	{
+        //print error 
+		exit(-1);
```

Comment:
Could this print a more descriptive error, and should it be SendErrorAndCrash instead of calling exit directly?

https://github.com/timothygrant80/cisTEM/blob/0aabb63c9cc6197150891606e85719f4ea9f57b7/src/core/myapp.cpp#L609-L614